code-challenges.js

```diff
@@ -14,36 +14,104 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+const longerString = (stringOne, stringTwo) => {
+  if(stringOne.length > stringTwo.length) {
+    return stringOne
+  } else {
+    return stringTwo
+  }
+}
+
+console.log(longerString(fruit1, fruit2))
+console.log(longerString(fruit3, fruit4))
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+// const temp1 = 42
+// const temp2 = 350
+// const temp3 = 212
+ 
+// Psuedo Code
+// a function that takes x and y and gives value of below, at, or above boiling point
+// will need an if/let statement
+// looking for 3 different outputs from given data
+
+
+// if(temp1 < 212) {
+//   console.log(`${temp1} is below boiling point`)
+// } else if(temp2 > 349) {
+//   console.log(`${temp2} is above boiling point`)
+// } else if(temp3 > 42 && temp3 < 349){
+//   console.log(`${temp3} is at boiling point`)
+// } else {
+//   console.log("please enter a number from 0 to 100")
+// }
```

Comment:
The if-else statement looks great!  You mentioned in your pseudocode using a function.  Where is the function, and how is it called?